_data/simple-icons.json

```diff
@@ -2766,6 +2766,11 @@
             "hex": "119EFF",
             "source": "https://github.com/ionic-team/ionicons-site/blob/b0c97018d737b763301154231b34e1b882c0c84d/docs/ionicons/svg/logo-capacitor.svg"
         },
+        {
+            "title": "CapRover",
+            "hex": "EF5C26",
```

Comment:
I'm not seeing this hex code in the (updated) source icon. I'd maybe go with `#ed5b26`, the color of the largest path in the center of the circle?

Reply:
Yes, you're right. Just updated the hex code.